src/utils/index.tsx

```diff
@@ -98,10 +102,17 @@ export async function getPathInfo(
     query TypenameAndCurrentPath($alias: AliasInput) {
       uuid(alias: $alias) {
         __typename
-        ... on User { alias }
-        ... on Page { alias }
-        ... on TaxonomyTerm { alias }
-        ... on AbstractEntity { alias }
+        ... on TaxonomyTerm {
+          alias
+          instance
+        }
+        ... on AbstractRepository {
+          alias
+          instance
+        }
+        ... on User {
+          alias
+        }
```

Comment:
Why is AbstractEntity left out here?

Reply:
Because every type implementing AbstractEntity also implements AbstractRepository